### PR TITLE
Improved the wording of the wait options

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala [hunk 1]
```diff
@@ -210,13 +210,12 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
 
     }
     section(title="Wait options", id="administration-wait-nowait", role = "enterprise-edition") {
-      p("""Aside from `SHOW DATABASES`, the database management commands all accept an optional
-          |`WAIT`/`NOWAIT` clause. The `WAIT`/`NOWAIT` clause allows a user to specify whether to wait
-          |for the command to complete before returning. The options are:
+      p("""Aside from `SHOW DATABASES`, all database management commands accept an optional
```
Comment (Lojjs) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala:213]: Actually this text is out-of-date. In 4.4 we added ALTER DATABASE which also does not have a WAIT clause. So while you are anyway at it, you might update this to 'Aside from `SHOW DATABASES` and `ALTER DATABASE`, all management ...' in 4.4 and dev.

**Review by Lojjs — COMMENTED** [review]
I think the new text sounds good, but I realised the first sentence is actually outdated as of 4.4. Feel free to fix it here or create a separate card

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala (hunk 1 above)
Comment (renetapopova) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala:213]: Thanks, @Lojjs! I'll update it after I cherry-pick it into 4.4.

**Comment by renetapopova** [comment]
@Lojjs, are you happy with this PR to be merged as it is in 4.3?

**Comment by Lojjs** [comment]
> @Lojjs, are you happy with this PR to be merged as it is in 4.3?

Yes for 4.3 it looks fine :)

**Review by Lojjs — APPROVED** [review]

**Pull request merged**